fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/OnboardingPageUiData.kt

```diff
@@ -13,6 +13,7 @@ import org.mozilla.fenix.nimbus.OnboardingCardData
 data class OnboardingPageUiData(
     val type: Type,
     @DrawableRes val imageRes: Int,
+    val imageIsIllustration: Boolean,
```

Comment:
I think we could name it `isContentScaleFit` as this is a purely ui model and we can refer to styling here. Or we can create enums `Fit` and `Crop`. wdyt?